typespec-tests/src/main/java/com/cadl/naming/models/RequestType.java

```diff
@@ -66,7 +66,7 @@ public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
 
     /**
      * Reads a RequestType from the JSON stream.
-     * &lt;p&gt;.
+     * <p>
```

Comment:
will need to add `@Generated` annotation to `fromJson` and `toJson`. Can be done in following pr.

Reply:
Will add this in a follow up PR